lib/main_common.pm

```diff
@@ -1662,15 +1662,20 @@ sub get_wicked_tests {
     return @tests;
 }
 
-sub load_wicked_barrier {
-    my $args = OpenQA::Test::RunArgs->new();
-    my @wicked_tests = get_wicked_tests(only_names => 1);
-    $args->{wicked_tests} = \@wicked_tests;
-    loadtest('wicked/barrier_init', run_args => $args);
+sub wicked_init_locks {
+    if (get_var('IS_WICKED_REF')) {
+        my $args = OpenQA::Test::RunArgs->new();
+        my @wicked_tests = get_wicked_tests(only_names => 1);
+        $args->{wicked_tests} = \@wicked_tests;
+        loadtest('wicked/locks_init', run_args => $args);
+    }
+    else {
+        loadtest('wicked/locks_init');
```

Comment:
this will require us calling quite heavy  `get_wicked_test` function without a good reason. And create empty `RunArgs` object. Or I misunderstood your idea ? 
Something like this and then decide inside locks_init ?
```    
    my $args = OpenQA::Test::RunArgs->new();
    my @wicked_tests = get_wicked_tests(only_names => 1);
    $args->{wicked_tests} = \@wicked_tests;
     loadtest('wicked/locks_init', run_args => $args);
```